<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCreateModelsModule</class>
 <widget class="qSlicerWidget" name="qSlicerCreateModelsModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="19" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Create models</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
        <property name="title">
         <string>Needle model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Length: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLSpinBox" name="NeedleLength" native="true">
           <property name="decimals" stdset="0">
            <number>1</number>
           </property>
           <property name="maximum" stdset="0">
            <double>5000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>80.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Depth markers: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="markersCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Radius: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qMRMLSpinBox" name="NeedleRadius" native="true">
           <property name="decimals" stdset="0">
            <number>1</number>
           </property>
           <property name="maximum" stdset="0">
            <double>500.000000000000000</double>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Tip radius:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="qMRMLSpinBox" name="NeedleTipRadius" native="true">
           <property name="maximum" stdset="0">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>2.500000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="CreateNeedleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create needleeeeee</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="SwitchModuleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Next step</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
        <property name="title">
         <string>Coordinate system model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="CreateCoordinateButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create coordinate system</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Axis length: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Thickness: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qMRMLSpinBox" name="CAxisThicknessSpinBox" native="true">
           <property name="maximum" stdset="0">
            <double>200.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>2.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLSpinBox" name="CAxisLengthSpinBox" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>20.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_3">
        <property name="title">
         <string>Cube model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="qMRMLSpinBox" name="CubeALength" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>R-L length: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="CreateCubeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create cube</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>S-I length: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>A-P length: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLSpinBox" name="CubeRLength" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qMRMLSpinBox" name="CubeSLength" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_4">
        <property name="title">
         <string>Cylinder model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="qMRMLSpinBox" name="CylinderHeight" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>40.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="CreateCylinderButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create cylinder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Height: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qMRMLSpinBox" name="CylinderRadius" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Radius: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_5">
        <property name="title">
         <string>Sphere model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="CreateSphereButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create sphere</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLSpinBox" name="SphereRadius" native="true">
           <property name="maximum" stdset="0">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="quantity" stdset="0">
            <string>length</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Radius:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSpinBox</class>
   <extends>ctkDoubleSpinBox</extends>
   <header>qMRMLSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CollapsibleButton</tabstop>
  <tabstop>CollapsibleGroupBox</tabstop>
  <tabstop>markersCheckBox</tabstop>
  <tabstop>CreateNeedleButton</tabstop>
  <tabstop>CollapsibleGroupBox_2</tabstop>
  <tabstop>CreateCoordinateButton</tabstop>
  <tabstop>CollapsibleGroupBox_3</tabstop>
  <tabstop>CreateCubeButton</tabstop>
  <tabstop>CollapsibleGroupBox_4</tabstop>
  <tabstop>CreateCylinderButton</tabstop>
  <tabstop>CollapsibleGroupBox_5</tabstop>
  <tabstop>CreateSphereButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>NeedleLength</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>NeedleRadius</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>NeedleTipRadius</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CAxisLengthSpinBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CAxisThicknessSpinBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CubeRLength</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CubeALength</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CubeSLength</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CylinderHeight</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CylinderRadius</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCreateModelsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SphereRadius</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
